### PR TITLE
all layouts in card

### LINKED ISSUE #14: Icons not aligned in card
icons breaking in some screen size, need to be in 1 line

### PR DESCRIPTION
**Pull request by shivanigangadharan**
Layouts -

- Cards with badges
- Cards with dismiss
- Cards with text overlay
- Text only cards
- Horizontal cards
- Cards with shadow

### CODE REVIEW

File: pages/docs/docs.html [hunk 1]
```diff
@@ -278,6 +278,50 @@ <h3> Text only cards</h3>
                             </div>
                         </div>
                     </div>
+                    <div>
+                        <h3>Horizontal card</h3>
+                        <div class="container-card">
+                            <div class="flex">
+                                <img class="sm-img" src="https://picsum.photos/id/237/100/100" />
```
Comment (vishalg8454) [inline pages/docs/docs.html]: use of `alt` attribute in image is recommended.

**Review by vishalg8454 — APPROVED** [review]
All fine, looks ready to merge.

File: pages/docs/docs.html (hunk 1 above)
Comment (shivanigangadharan) [reply pages/docs/docs.html]: Done in commit [1e2ce5a](https://github.com/shivanigangadharan/style-ui/pull/15/commits/1e2ce5ae9cacc76918a8abb4a28e03d813999e7b)

**Review by SanjayJatti — APPROVED** [review]
icons in footer of card are breaking. Otherwise all good.

**Comment by shivanigangadharan** [comment]
> icons in footer of card are breaking. Otherwise all good.

Tried to fix in commit [b6a2db3](https://github.com/shivanigangadharan/style-ui/pull/15/commits/b6a2db33dae60131e4ee82d4fb493eee2b8e9900).

**Pull request merged**